projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/blogroll-item.php

```diff
@@ -75,7 +75,7 @@ function load_assets( $attr, $content, $block ) {
 
 	$placeholder_site_icon = '';
 	$site_icon_html        = <<<HTML
-	<img src="$icon" alt="$name_attr" onerror="this.parentNode.classList.add('empty-site-icon')">
+	<img id="blogroll-item-image" src="$icon" alt="$name_attr" onerror="this.parentNode.classList.add('empty-site-icon')">
```

Comment:
Why set an id here? We can have multiple items and thus multiple images with the same id.

Reply:
@heavyweight - Updated this to use a class instead. This is used for css specificity. If we only target the `img` tag, theres other CSS (`editor-styles-wrapper`) thats overriding the height of the image.